scripts/run_lighthouse_tests_test.py

```diff
@@ -82,6 +82,9 @@ def mock_print(msg: str) -> None:
                 LIGHTHOUSE_CONFIG_FILENAMES[LIGHTHOUSE_MODE_PERFORMANCE]['1']),
             '--max-old-space-size=4096'
         ]
+        # Arguments to record in lighthouse_setup.js.
+        self.extra_args = ['-record', os.path.join(
+                os.getcwd(), '..', 'lhci-puppeteer-video', 'video.mp4')]
```

Comment:
```suggestion
        self.extra_args = [
            '-record', 
            os.path.join(os.getcwd(), '..', 'lhci-puppeteer-video', 'video.mp4')
        ]
```

When the elements inside parentheses/brackets/braces need to be split over multiple lines, we always start with a newline right after the opening parenthesis/bracket/brace. See style guide for details

Reply:
Updated code to reflect suggestion in new commit.